linux_docker_resources/Dockerfile

```diff
@@ -61,23 +61,31 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
   python3-pyparsing \
   python3-pytest-mock \
   python3-yaml \
-  uncrustify
+  uncrustify \
+  libgazebo9-dev \
```

Comment:
Note that Noetic and Foxy should use `gazebo11`

Reply:
Do you think it's worth maintaining a ros2/ci branch for Gazebo 9? I guess we can rely on the build.ros2.org CI jobs for released distros (like Dashing and Eloquent).

Reply:
Good point, we only need 11 at this point :+1: